event-consumer/manifest.yml

```diff
@@ -0,0 +1,35 @@
+packages:
+  __APP_PACKAGE__:
+    license: Apache-2.0
+    actions:
+      generic:
+        function: actions/generic/index.js
+        web: 'yes'
+        runtime: 'nodejs:12'
+        inputs:
+          LOG_LEVEL: debug
+          apiKey: $SERVICE_API_KEY
+          providerId: $PROVIDER_ID
+          integration_id: $EVENT_INTEGRATION_ID
+          eventCode: $EVENT_CODE
+          client_id: $CLIENT_ID
+          client_secret: $CLIENT_SECRET
+          technical_account_email: $TECH_ACCOUNT_EMAIL
+          technical_account_id: $TECH_ACCOUNT_ID
+          ims_org_id: $IMS_ORG_ID
+          private_key: $PRIVATE_KEY
+          journalling_url: $JOURNALLING_URL
+          db_event_key: $DB_EVENT_KEY
+          max_events_in_batch: $MAX_EVENTS_IN_BATCH
```

Comment:
Just curious here. Do you get all these values `$CLIENT_ID`, `$CLIENT_SECRET` from the console and then you set them manually in your `.env` file ?

Reply:
+1, adding sample dot-env file would make it easier to understand.

Reply:
done